src/mixins/graphql.js

```diff
@@ -55,11 +60,14 @@ export default {
     /**
      * GraphQL query variables.
      *
-     * @returns {{workflowId: string}}
+     * @returns {{workflowId: string, workflowName: string}}
      */
     variables () {
       return {
-        workflowId: this.workflowId
+        workflowId: this.workflowId,
+        workflowName: this.workflowName,
+        task: '',
+        file: ''
```

Comment:
The unit tests are failing due to this change. Not sure if I should be fixing the test or doing this assignment in another place? Will wait for review opinion.